ports/psoc6/boards/CY8CPROTO-062-4343W/makefile_mtb.mk

```diff
@@ -42,7 +42,7 @@ mpy_mtb_init:
 
 
 # build MTB project
-mpy_mtb_build:
+mpy_mtb_build: mpy_mtb_init
```

Comment:
I agree. Adding _mpy_mtb_init_, as pre-requisite for the _mpy_mtb_build_ target is unnecessary since the default make target of the main Makefile already invokes the build/compilation for all files (mtb and mpy both). 

My thinking was that if somehow someone invokes _mpy_mtb_build_ directly (without running _mpy_mtb_init_), then it would throw an error. Adding the pre-requisite will call the init recursively and then build, not throwing an error. Let me know what you think.

Reply:
`mpy_mtb_init` calls `make getlibs `from the MTB, which is required when we start working with the port, or when we want to update any of the dependencies of the board. 
Best case, this command will every time check that the versions of the board dependencies and those available in the mtb_shared folder are matching. 
So even unnecessary for subsequent builds, probably it is not really slowing things down much. 
Personally, it is more about hidden behavior... `make` will not only build, but update modules and dependencies (in worst case). 
You can claim that usually a `program` target is also building prior flashing... But being this one a bit less conventional and a one time/occasional action in the development flow I would not keep it as pre-requisite.

Reply:
Agreed.